### ISSUE TITLE
leptosfmt adds tow newlines at the end of a file

### ISSUE THREAD

**Issue by TheBlckbird** [bram209/leptosfmt#94]
If you format a file in VSCode using `rustfmt` and this setting in the workspace:
```json
"rust-analyzer.rustfmt.overrideCommand": [
    "leptosfmt",
    "--stdin",
    "--rustfmt"
]
```

It adds two newlines at the end of a file.
The cause for this is that `leptosfmt` and `rustfmt` both add a newline.

**Comment by koopa1338** [comment]
I think this is a duplicate of #92

**Comment by bram209** [comment]
Thanks for reporting!

Closing, as @koopa1338  stated, this is a duplicate of #92 (which is currently being addressed in #95).
